src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -102,13 +111,19 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                     .WithHandler<RazorConfigurationEndpoint>()
                     .WithHandler<RazorFormattingEndpoint>()
                     .WithHandler<RazorSemanticTokensEndpoint>()
+                    .AddHandlerLink("textDocument/semanticTokens/full/delta", LanguageServerConstants.LegacyRazorSemanticTokensEditEndpoint)
```

Comment:
Should come from constants

Reply:
Is this a new expansion of the semantic token spec?

Reply:
They changed the name of `textDocument/semanticTokens/edits` request to `textDocument/semanticTokens/full/delta` based on comments that the name was confusing and it wasn't clear that you can only request deltas on a full document and not a range.

VS has yet to react to this change in the spec, but we'll probably (unofficially) support both method names for the short term until all partners react.

I've filed a bug for this: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1181012